subt_ign/scripts/subt_docker_list.bash

```diff
@@ -25,7 +25,15 @@ fi
 
 # Log into docker
 echo "Logging into docker"
-$(aws ecr get-login --no-include-email --region us-east-1)
+awsVersion=`aws --version`
+if [[ "$awsVersion" == *"aws-cli/1"* ]]; then
+  $(aws ecr get-login --no-include-email --region us-east-1)
+elif [[ "$awsVersion" == *"aws-cli/2"* ]]; then
+  aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 200670743174.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
This variant works for me with `aws-cli/1.18.157 Python/3.6.9 Linux/4.15.0-123-generic botocore/1.18.16` so maybe the old one is not needed at all.